<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Methodist</class>
 <widget class="QWidget" name="Methodist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="feedback_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>40</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
       </property>
       <property name="text">
        <string>Обратная связь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prepare_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
       </property>
       <property name="text">
        <string>Подготовка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="study_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
       </property>
       <property name="text">
        <string>Обучение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="homepage_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #008cba;
	border: 0 none;
    color: white;
	width: 100%;
}

QPushButton:hover {
    background-color: #006687;
}</string>
       </property>
       <property name="text">
        <string>Личный кабинет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="showing_selector_">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
